e2e/addresses_test.go

```diff
@@ -42,16 +42,16 @@ func TestAddresses(t *testing.T) {
 
 	t.Logf("addr1 = %s", fmt.Sprint(addr1))
 
-	t.Cleanup(func() { theCloud.Addresses().Delete(context.Background(), addr1Key) })
+	t.Cleanup(func() { TheCloud.Addresses().Delete(context.Background(), addr1Key) })
```

Comment:
Please consider not changing the variable name. It doesn't seem needed (I run `sed` and the code still worked). It would make PR diff much smaller

Reply:
reverted to `theCloud`